sound/soc/intel/common/soc-acpi-intel-icl-match.c

```diff
@@ -29,6 +28,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_icl_machines[] = {
 		.sof_fw_filename = "sof-icl.ri",
 		.sof_tplg_filename = "sof-icl-rt5682.tplg",
 	},
+	{
+		.id = "DLGS7219",
+		.drv_name = "jsl_da7219_max98373",
```

Comment:
can't you play with aliases and use icl_da7219_max...

Reply:
Ack.